.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
please remove the IDE settings

Reply:
sorry for this one... I always expect this to be in .gitignore... will add ;-)